components/gcp/dataproc/delete_cluster/component.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: dataproc_delete_cluster
+description: |
+  Deletes a DataProc cluster.
+inputs:
+  - name: project_id
```

Comment:
Can we later get the project ID using wget on the metadata endpoint?

Reply:
I think even we support the project id from metadata endpoint. It will be supported either in the notebook sample or as the default value in component code if the value is not provided. The component should still provide project ID parameter as user might want to run the job in another project.

Reply:
make sense to expose the project ID but having a default project ID as the current project helps.

Reply:
Yes, it will be added in the future.

Reply:
Great.
/lgtm